umap/umap_.py

```diff
@@ -2671,7 +2727,7 @@ def fit_transform(self, X, y=None):
         r_emb: array, shape (n_samples)
             Local radii of data points in the embedding (log-transformed).
         """
-        self.fit(X, y)
+        self.fit(X, y, pin_mask)
```

Comment:
Probably best to make this a keyword arg in case we get more fit parameters later (such as a ``init`` fit param).

```suggestion
        self.fit(X, y, pink_mask=pin_mask)
```